units/generate_tasks.go

```diff
@@ -156,21 +156,26 @@ func (j *generateTasksJob) generate(ctx context.Context, t *task.Task) error {
 }
 
 // handleError return mongo.ErrNoDocuments if another job has raced, the passed in error otherwise.
-func (j *generateTasksJob) handleError(v *model.Version, handledError error) error {
+func (j *generateTasksJob) handleError(pp *model.ParserProject, v *model.Version, handledError error) error {
 	if v == nil {
 		return handledError
 	}
 	versionFromDB, err := model.VersionFindOne(model.VersionById(v.Id).WithFields(model.VersionConfigNumberKey))
 	if err != nil {
 		return errors.Wrapf(err, "problem finding version %s", v.Id)
 	}
+	ppFromDB, err := model.ParserProjectFindOne(model.ParserProjectById(v.Id).WithFields(model.ParserProjectConfigNumberKey))
+	if err != nil {
+		return errors.Wrapf(err, "problem finding version %s", v.Id)
```

Comment:
nit: "problem finding parser project" (also can this check be moved after the versionFromDB == nil check? Mildly confusing in this order)